core/lib/protobuf_config/src/proto/fri_prover_group.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package zksync.config;
+
+message CircuitIdRoundTuple {
+  optional uint32 circuit_id = 1; // required; u8
+  optional uint32 aggregation_round = 2; // required; u8
+}
+
+message FriProverGroup {
+  repeated CircuitIdRoundTuple group_0 = 1;
```

Comment:
Maybe repeated FriProverGroup or map over CircuitIdRoundTuple would be better.
AFAIK number of groups could change.